index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <header class="header_container">
+        <a href="#"><h2>Welcome</h2></a> 
+        <nav class="navbar">
+            <img src="/assets/Union.png" alt="">
+        </nav>
+    </header>
+    <main>
+      <section class="intro_section">
+        <h1> Hey there. <br> I'm Favour <br>
+            <span>I'm a software developer</span>
+        </h1>
+        <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        </section>
+        <ul>
+            <li>
+                <a href="www.facebook.com"><i class="fa fa-facebook" aria-hidden="true"></i></a>
+            </li>
+            <li>
+                <a href="twitter.com/Fabimworld2536"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li>
+                <a href="www.github.com/fayob"><i class="fa fa-github" aria-hidden="true"></i></a>
+            </li>
+            <li>
+                 <a href="www.linkedin.com/in/abimbola-ade"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+        </ul>
```

Comment:
- [ ] [OPTIONAL] Great job so far I would like to recommend you use the same `grey` and also the size for your social media icons as defined on the template :+1: because Follow the original vision if it's pretty and works, it will be a wonderful website.